.github/workflows/docker-build-and-run.yml

```diff
@@ -98,8 +98,8 @@ jobs:
 
       - name: Verify input files
         run: |
-          IVIM_SIM_DIR = "${{ github.workspace }}/ivim_simulation"
-          for file in ivim_simulation_0000.dcm, ivim_simulation_0001.dcm, ivim_simulation_0002.dcm ivim_simulation_0003.dcm ivim_simulation_0004.dcm; do
+          IVIM_SIM_DIR="${{ github.workspace }}/ivim_simulation"
```

Comment:
Still failing. https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/actions/runs/15230335594/job/42931601286?pr=99
Probably not quite the right path, especially the comma at the end.
You should be able to test this on your end by running the actions rather than waiting for me to run and check the tests.